### ISSUE TITLE
data format

### ISSUE THREAD

**Issue by mistersherlock** [zizhaozhang/unet-tensorflow-keras#15]
In train directory
train/
            img/
                0/
            gt/
                0/
in 0/ need I to put one image or batch?
In my case, I have 6 classes. How to look groundtruth mask image?  0,1,2,3,4,5 in the last channel, in all channels, certain colors like in palette_refer.png

**Comment by NXTNiklas** [comment]
Got the same problem! Did you solve it?

I'm exspecially confused of the folders in img. Because i have multiple classes on one Image..

**Comment by ak3ra** [comment]
What is the correct folder structure for the data necessary to run this model?
